lib/burp/html/issue.rb

```diff
@@ -153,7 +153,7 @@ def summary
         table = h2.next_element
 
         summary_table_tags.each do |tag|
-          td = table.search("td:starts-with('#{tag.to_s.capitalize}:')").first
+          td = table.xpath("//td[text()[starts-with(.,'#{tag.to_s.capitalize}:')]]").first
```

Comment:
How is this different from:

```suggestion
          td = table.xpath("//td[starts-with(.,'#{tag.to_s.capitalize}:')]").first
```

Reply:
good observation. it behaves the same way. changed it!